docs/appnotes/README.md

```diff
@@ -0,0 +1,22 @@
+# Application Notes
+
+This folder contains applciation notes in markdown format.
+These documents provide ancillary information about the concepts in the TAMS API, and their application to real-world problems.
+No part of these documents are considered part of the API specification, although including specification-type language as examples, or "opt-in" extensions is allowed.
+
+There is a simple template `appnote-template.md` which describes the few restrictions there are on Application Notes.
+Select a unique number for your Applcation Note, and try to keep them at least somewhat close together.
```

Comment:
It turns out the way we number the ADRs is mildly annoying, in that it's easy to get conflicts. But I can't think of any elegant solutions to that: naming based on the date is either differently annoying, or doesn't fix the problem!